docs-src/docs/prerequisites-for-annotation.mdx

```diff
@@ -0,0 +1,62 @@
+---
+id: prerequisites-for-annotation
+title: Prerequisites for Annotation
+slug: /prerequisites-for-annotation
+---
+
+import Tabs from "@theme/Tabs";
+import TabItem from "@theme/TabItem";
+
+# Prerequisites for Annotation
+
+Before scenes can be annotated a few things need to be in place. From a developer perspective there is not too much to do;
+this is mostly organization and workforce level preparation, but it is good to be aware of the process.
+
+<details>
+  <summary>
+    <h2 style={{margin: '0px'}}>Projects and Requests</h2>
+  </summary>
+
+A scene is sent for annotation by adding it to a *Request*, creating what we call and *Input*, which has its own UUID.
+This is detailed on the next page.
+
+The request is part of a *Batch*, itself part of a *Project*. Projects and initial Requests and Batches will be set up,
+possibly in conjunction with Perception Experts from Kognic, during the onboarding/setup phase for new customers.
+They may also be managed independently or in conjunction with a BPO who provides annotation services through the Kognic platform.
+</details>
+
+<details>
```

Comment:
Annotation types will be removed so maybe we should just start removing them from the documentation?